CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# How to contribute
+
+As of 2020-09-13, Ross Light, the primary maintainer, does not have enough time
```

Comment:
I think this paragraph can be removed.  Perhaps we should add an official list of maintainers at the top of this file, namely:  myself, Ian and Taylor ?